protocol/protobuf/community_update.pb.go

```diff
@@ -417,24 +417,76 @@ func (m *ChannelData) GetChannel() *CommunityChat {
 	return nil
 }
 
+type SignedCommunityEvent struct {
+	// Signature of the payload field
+	Signature []byte `protobuf:"bytes,1,opt,name=signature,proto3" json:"signature,omitempty"`
+	// Marshaled CommunityEvent
+	Payload              []byte   `protobuf:"bytes,2,opt,name=payload,proto3" json:"payload,omitempty"`
+	XXX_NoUnkeyedLiteral struct{} `json:"-"`
+	XXX_unrecognized     []byte   `json:"-"`
+	XXX_sizecache        int32    `json:"-"`
+}
+
+func (m *SignedCommunityEvent) Reset()         { *m = SignedCommunityEvent{} }
+func (m *SignedCommunityEvent) String() string { return proto.CompactTextString(m) }
+func (*SignedCommunityEvent) ProtoMessage()    {}
+func (*SignedCommunityEvent) Descriptor() ([]byte, []int) {
+	return fileDescriptor_52ed23dfc73918ab, []int{4}
+}
+
+func (m *SignedCommunityEvent) XXX_Unmarshal(b []byte) error {
+	return xxx_messageInfo_SignedCommunityEvent.Unmarshal(m, b)
+}
+func (m *SignedCommunityEvent) XXX_Marshal(b []byte, deterministic bool) ([]byte, error) {
+	return xxx_messageInfo_SignedCommunityEvent.Marshal(b, m, deterministic)
+}
+func (m *SignedCommunityEvent) XXX_Merge(src proto.Message) {
+	xxx_messageInfo_SignedCommunityEvent.Merge(m, src)
+}
+func (m *SignedCommunityEvent) XXX_Size() int {
+	return xxx_messageInfo_SignedCommunityEvent.Size(m)
+}
+func (m *SignedCommunityEvent) XXX_DiscardUnknown() {
+	xxx_messageInfo_SignedCommunityEvent.DiscardUnknown(m)
+}
+
+var xxx_messageInfo_SignedCommunityEvent proto.InternalMessageInfo
+
+func (m *SignedCommunityEvent) GetSignature() []byte {
+	if m != nil {
+		return m.Signature
+	}
+	return nil
+}
+
+func (m *SignedCommunityEvent) GetPayload() []byte {
+	if m != nil {
+		return m.Payload
+	}
+	return nil
+}
+
 // CommunityEventsMessage is a message used to propagate information
 // about community changes.
 type CommunityEventsMessage struct {
 	CommunityId []byte `protobuf:"bytes,1,opt,name=community_id,json=communityId,proto3" json:"community_id,omitempty"`
 	// Events base CommunityDescription with owner signature on top of which events were generated
 	EventsBaseCommunityDescription []byte `protobuf:"bytes,2,opt,name=events_base_community_description,json=eventsBaseCommunityDescription,proto3" json:"events_base_community_description,omitempty"`
 	// A list of admins events for the channel in bytes
-	Events               [][]byte `protobuf:"bytes,3,rep,name=events,proto3" json:"events,omitempty"`
-	XXX_NoUnkeyedLiteral struct{} `json:"-"`
-	XXX_unrecognized     []byte   `json:"-"`
-	XXX_sizecache        int32    `json:"-"`
+	// Deprecated: use signed_events instead.
```

Comment:
I'm not sure that it is correct to edit pb.go files

Reply:
This is auto-generated thanks to `[deprecated=true]` tag.